vault/mount.go

```diff
@@ -1487,6 +1478,22 @@ func (c *Core) setupMounts(ctx context.Context) error {
 			}
 		}
 
+		// Do not start up deprecated builtin plugins. If this is a major
+		// upgrade, stop unsealing and shutdown. If we've already mounted this
+		// plugin, skip backend initialization and mount the data for posterity.
```

Comment:
I don't think "mount the data" is a thing.

Reply:
Suggestion for improvement?